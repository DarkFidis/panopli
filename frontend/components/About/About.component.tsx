import React from 'react';
import styles from '../../styles/About.module.css'

export const About: React.FC = () => (
  <section id="about" className={styles.about}>
    <h1 className="heading"><span>a</span> <span>p</span>ropos</h1>
    <div className={styles.row}>
      <div className={styles.image}>
        <img src="/images/about-img.png" alt=""/>
      </div>
      <div className={styles.content}>
        <h3>Qui sommes nous ?</h3>
        <p>Matthieu Seynaeve, ancien cartographe reconverti développeur web, vous propose un petit service simple, afin que vous puissiez trouver un bon restaurant
        près de votre emplacement. Cette application repose sur du requêtage de proximité basé sur Mongo DB ainsi que Leaflet JS pour la partie cartographie</p><br/>
        <p>A vous de trouver votre bonheur !</p>
        <a href="https://darkfidis.github.io/portfolio/" target='_blank' rel="noopener noreferrer">
          <button className={styles.btn}>En savoir plus</button>
        </a>
      </div>
    </div>
  </section>
)

export default About
