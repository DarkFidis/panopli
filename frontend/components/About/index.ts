import AboutComponent, { About } from "./About.component";
export { About }
export default AboutComponent
