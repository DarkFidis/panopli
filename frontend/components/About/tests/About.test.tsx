import * as React from 'react'

import { render } from "@testing-library/react";
import About from "../index";

describe('About component unit test', () => {
  it('should match snapshot', () => {
      const { container } = render(<About />)
      expect(container).toMatchSnapshot()
  })
})
