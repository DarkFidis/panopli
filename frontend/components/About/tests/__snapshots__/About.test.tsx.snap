// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component unit test should match snapshot 1`] = `
<div>
  <section
    class="about"
    id="about"
  >
    <h1
      class="heading"
    >
      <span>
        a
      </span>
       
      <span>
        p
      </span>
      ropos
    </h1>
    <div
      class="row"
    >
      <div
        class="image"
      >
        <img
          alt=""
          src="/images/about-img.png"
        />
      </div>
      <div
        class="content"
      >
        <h3>
          Qui sommes nous ?
        </h3>
        <p>
          Matthieu Seynaeve, ancien cartographe reconverti développeur web, vous propose un petit service simple, afin que vous puissiez trouver un bon restaurant près de votre emplacement. Cette application repose sur du requêtage de proximité basé sur Mongo DB ainsi que Leaflet JS pour la partie cartographie
        </p>
        <br />
        <p>
          A vous de trouver votre bonheur !
        </p>
        <a
          href="https://darkfidis.github.io/portfolio/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <button
            class="btn"
          >
            En savoir plus
          </button>
        </a>
      </div>
    </div>
  </section>
</div>
`;
